test/components/converters/test_tika_doc_converter.py

```diff
@@ -18,16 +18,18 @@ def test_run(self, mock_tika_parser):
         assert len(documents) == 1
         assert documents[0].content == "Content of mock source"
 
-    def test_run_with_meta(self):
+    def test_run_with_meta(self, test_files_path):
```

Comment:
This test has become an integration test. Right?

Reply:
No, Tika itself is mocked, but the converter does load the file so I'm pointing to a real file here